tools/c7n_gcp/tests/test_sql.py

```diff
@@ -36,8 +37,8 @@ def test_sqlinstance_get(self):
              'resource': 'gcp.sql-instance'},
             session_factory=factory)
         instance = p.resource_manager.get_resource(
-            {"project": "cloud-custodian",
-             "name": "brenttest-2"})
+            {'project_id': 'cloud-custodian',
+             'database_id': 'cloud-custodian:brenttest-2'})
```

Comment:
could clarify this is actually in the event for api calls?

Reply:
Yes. It is. Here is excerpt from a log entry:
```json
{
  "protoPayload": {
    ...
     "methodName": "cloudsql.instances.create",
     "resourceName": "instances/mitrop-custodian:test-mysql",
    ...
  },
   "resource": {
     "type": "cloudsql_database",
     "labels": {
       "project_id": "mitrop-custodian",
       "database_id": "mitrop-custodian:test-mysql",
       "region": "us-central"
     }
   },
   ...
}
```

Reply:
thanks for confirming.